app/views/layouts/_flash_messages.html.haml

```diff
@@ -0,0 +1,4 @@
+- flash.each do |type,msg|
+  .alert{ class: "alert-#{bootstrap_flash_type(type)}" }
```

Comment:
need to bring in the `boostrap_flash_type` helper method into the application helper module